index.html

```diff
@@ -18,7 +18,10 @@
   <div id="ui">
     <div id="header">
       <div id="title">
-        <h3>Pitt Open Data Project</h3>
+        <div id="logo">
```

Comment:
Also we lose both Home and About nav links on mobile now, so this would make sure we have enough room.